commands/user.bee.inc

```diff
@@ -67,12 +151,16 @@ function users_bee_callback($arguments, $options) {
     if ($user->uid == 0) {
       continue;
     }
-
+    $roles = '';
+    foreach($user->roles as $role){
+      $roles = $roles.$role.', ';
+    }
```

Comment:
A concatenated string should have spaces between the strings and the the dot. Also, control structures should have a space between the control word and the condition brackets and the opening braces (not sure why the phpcs check is not picking this up)
```suggestion
    foreach ($user->roles as $role) {
      $roles = $roles . $role . ', ';
    }
```
Also, it would be good to:
- exclude `authenticated` - every user in this list will have it.
- not include the trailing comma at the end of the list.